layouts/partials/comments/provider/giscus.html

```diff
@@ -1,3 +1,7 @@
+{{ $lang := .Language.LanguageCode }}
+{{ if not $lang }}
+    {{ $lang = .Language.Lang }}
```

Comment:
According to https://gohugo.io/content-management/multilingual/, `.Language.Lang` doesn't seem to be a public variable, only `.Language.LanguageCode`. 

Then this change can be simplified as `data-lang="{{ default $.Language.LanguageCode .lang }}"` I think

Reply:
Thank you for your review and feedback. I've created a new pull request (#1075) that only includes the changes about giscus。
And about the `.Language.Lang` variable, you can see https://gohugo.io/methods/page/language/